app/views/keywords/show.html.slim

```diff
@@ -0,0 +1,47 @@
+div class="d-flex flex-column align-items-center mt-4"
+
+  h1.display-3 == keyword.keyword
+
+  - if keyword.status == 'completed'
+    div class="d-flex mt-5 w-100 justify-content-center"
+      div id="topPosAdwordsCount" class="card"
+        div class="card-body" 
+          .display-4 = keyword.top_pos_adwords
+        div class="card-info" = t('keyword.top_pos_adwords').upcase
```

Comment:
instead of using Ruby `upcase` can we use CSS to make that 🤔

Reply:
Fixed in 2bf1fa4a09a6ffee5b4ef7bc7d9ec24383eed9f6